common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -54,7 +54,6 @@
         border-bottom: 0.0625rem dotted #dfdfdf;
```

Comment:
yep, you can tell git to ignore permissions changes with `git config core.fileMode false`